src/lib/requestManager.ts

```diff
@@ -20,14 +23,29 @@ export const defaultOptions: AxiosRequestConfig = {
  *                   an error object for its reason
  */
 
+const attachAuthorizationHeader = (requestOptions: AxiosRequestConfig) => {
+  const tokens = getLocalStorageValue(LocalStorageKey.tokens);
+
+  if (tokens) {
+    requestOptions.headers = {
+      ...requestOptions.headers,
+      authorization: `Bearer ${(tokens as Tokens).accessToken}`,
```

Comment:
How about cast the variable in the assignment? it would be easier to read.
```suggestion
  const tokens = getLocalStorageValue(LocalStorageKey.tokens) as Tokens;

  if (tokens) {
    requestOptions.headers = {
      ...requestOptions.headers,
      authorization: `Bearer ${tokens.accessToken}`,
```